src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,150 @@
+package duke.main;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Duke {
+    static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+
+        String userInput;
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        printOutput("Hello! I'm duke.main.Duke\n" +
+                "What can I do for you?", true);
+        while (true) {
+            userInput = in.nextLine();
+            if (userInput.equalsIgnoreCase("bye")) {
+                break;
+            } else if (userInput.equalsIgnoreCase("list")) {
+                listTasks(userInput);
+            } else if (userInput.toLowerCase().startsWith("done ")) {
+                setDone(userInput);
+            } else if (userInput.toLowerCase().startsWith("deadline")) {
+                createDeadline(userInput);
+            } else if (userInput.toLowerCase().startsWith("event")) {
+                createEvent(userInput);
+            } else if (userInput.toLowerCase().startsWith("todo")) {
+                createTodo(userInput);
+            } else {
+                printOutput("Commands: bye list todo deadline event", false);
+            }
+        }
+        printOutput("Bye. Hope to see you again soon!", true);
+    }
+
+    private static void listTasks(String userInput) {
+        if (taskList.size() == 0) {
+            printOutput("Oops the list is empty!", false);
+            return;
+        }
+        String tl = "";
```

Comment:
The variable `tl` here is not very informative. You might want to give it a more informative naming.